go/ql/lib/semmle/go/Types.qll

```diff
@@ -740,6 +753,31 @@ class InterfaceType extends @interfacetype, CompositeType {
     exists(int i | i >= 0 | component_types(this, i, name, result))
   }
 
+  /**
+   * Gets the type of method `id` of this interface type.
+   *
+   * This differs from `getMethodType` in that if the method is not exported, the `id`
+   * will be package-qualified. This means that the set of `id`s` together with any
+   * embedded types fully distinguishes the interface from any other, whereas the set
+   * of names matched by `getMethodName` may be ambiguous between interfaces with matching
+   * exported methods and unexported methods that have matching names but belong to
+   * different packages.
+   *
+   * For example, `interface { Exported() int; notExported() int }` declared in two
+   * different packages defines two distinct types, but they appear identical according to
+   * `getMethodType`.
```

Comment:
It may be good to extend the example to include the `id`s of the methods (with sample packages) to show explicitly what you mean in the previous paragraph.